PluginGenerator/Maven/MavenArtifactHandlerExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MavenArtifactHandlerExtensions.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+
+namespace SonarQube.Plugins.Maven
+{
+    /// <summary>
+    /// Extension methods for IMavenArtifactHandler
+    /// </summary>
+    public static class MavenArtifactHandlerExtensions
```

Comment:
The methods in this class were only in the test code before. I've moved them to the product code so they can be used by test and product.